Languages/English/Keyed/Keys.xml

```diff
@@ -19,8 +19,8 @@
 	<Numbers_LoadDefault>Load default view</Numbers_LoadDefault>
 
 	<Numbers_ShowMoreInfoThanVanilla>Show needs and jobs even where it would normally be hidden.</Numbers_ShowMoreInfoThanVanilla>
-	<Numbers_coolerThanTheWildlifeTab>Numbers! is everything I ever wanted from the "Wildlife" tab and I want it to take over.</Numbers_coolerThanTheWildlifeTab>
-	<Numbers_MaxTableHeight>Maximum height of the "Numbers" tab that is allowed to use.</Numbers_MaxTableHeight>
+	<Numbers_coolerThanTheWildlifeTab>Numbers is everything I ever wanted from the "Wildlife" tab and I want it to take over.</Numbers_coolerThanTheWildlifeTab>
+	<Numbers_MaxTableHeight>Set the maximum screen height that is allowed to expand the "Numbers" tab.</Numbers_MaxTableHeight>
```

Comment:
How about "Set the maximum proportional screen height of the "Numbers" Tab".

Too complicated?

"Maximum size of the "Numbers" window relative to your screensize"? I think some like that works. What do you think?